03-viz.Rmd

```diff
@@ -290,6 +290,151 @@ faithful_scatter <- ggplot(faithful, aes(x = waiting, y = eruptions)) +
+
+```{r english-mother-tongue}
+can_lang %>%
+  filter(language == "English" | language == "French")
+```
+> `|` is a logical operator representing "OR", which allows us to filter the rows where the language column is equal to "English" OR equal to "French". 
+
+To answer our question, we will need to adjust the scale of the x and y axes so that they are on a log scale:
+
+```{r mother-tongue-vs-most-at-home-scale, fig.width=5.75, fig.height=5, warning=FALSE, fig.cap = "Scatter plot of number of Canadians reporting a language as their mother tongue vs the primary language at home with log adjusted x and y axes"}
+ggplot(can_lang, aes(x = most_at_home, y = mother_tongue)) +
+  geom_point() +
+  xlab("Language spoken most at home (number of Canadian residents)") +
+  ylab("Mother tongue (number of Canadian residents)") +
+  scale_x_log10(labels = scales::comma) +
+  scale_y_log10(labels = scales::comma)
+```
+
+From the visualization in Figure \@ref(fig:mother-tongue-vs-most-at-home-scale), we see that for the 214 languages in this data set, as the number of people who have a language as their mother tongue increases, so does the number of people who speak that language at home. When we see two variables do this, we call this a *positive relationship*. Because the points are fairly close together, we can say that the relationship is strong. Because drawing a straight line through these  points would fit the pattern we observe quite well, we say that it's linear.
+
+```{r changing_the_units, include = FALSE}
+english_mother_tongue <- can_lang %>%
+  filter(language == "English") %>%
+  pull(mother_tongue)
+census_popn <- 35151728
+```
+
+We saw that `r format(english_mother_tongue, scientific = FALSE, big.mark = ",")` people reported that their mother tongue was English in the 2016 Canadian census, but what does this number really mean? To really understand this number, we need context. In particular, how many people were in Canada when this data was collected? From the 2016 Canadian census profile, we can see that the population was reported to be `r format(census_popn, scientific = FALSE, big.mark = ",")` people. The number of people who report that English is their mother tongue is much more meaningful when we report it in this context. We can even go a step further and transform this count to a relative frequency or proportion to represent this as a single meaningful number in our data visualizations. We can do this by dividing the number of people reporting a given language as their mother tongue by the number of people who live in Canada. For example, the proportion of people who reported that their mother tongue was English in the 
```

Comment:
Can we do percentage instead of proportion? I think that is generally more understandable to a wider audience, especially when the proportion values are so small as they are here.